### DISCUSSION TITLE
Sidebar Hierarchy

### DISCUSSION THREAD

**Discussion by eli-s-goldberg** [observablehq/framework#1864] [Q&A]
I'm curious to add another layer of depth to the sidebar 
Something like this: 

```text
index/
├── Overview_dashboard/
├── Imaging/
│   ├── Analytics/
│   │   ├── Problem_Definition/
│   │   ├── EDA_ROI/
│   │   ├── Hypothesis/
│   │   ├── etc/
│   ├── Marketing/
│       ├── Collateral/
│       ├── etc/

```

Is this behavior supported by the observable.config.js? Is there a work around?

**Comment by Martien** [comment]
Not sure if this is the way to go. On the other hand, if it is generic, any number of layers can be supported. May cause usability and/or layout issues.

I am currently displaying layers deeper than 2 like so on the page: http://aarde.aardrock.nl/aardbron.

Generated by:
- https://github.com/Martien/Aarde/blob/main/src/aardbron.md
- https://github.com/Martien/Aarde/blob/main/src/zones/aardbron.js

**Reply by eli-s-goldberg** [reply]
Ah. I see. Interesting. That's certainly a hyperlinking work around. Thank you for writing!

I think I'd really like to see an 1-3 additional hierarchical layers of organization in the sidebar.
